helix-front/client/app/dashboard/dashboard.component.ts

```diff
@@ -169,7 +170,7 @@ export class DashboardComponent implements OnInit, AfterViewInit, OnDestroy {
       .getAll(this.clusterName)
       .subscribe(
         result => {
-          _.forEach(result, (resource) => {
+          lodashForEach(result, (resource) => {
```

Comment:
a workaround for this TypeScript related error: https://stackoverflow.com/q/49256040/1732222